docs/configs/janusgraph-cfg.md

```diff
@@ -152,7 +152,7 @@ Configuration options for HTTP(S) authentication.
 
 | Name | Description | Datatype | Default Value | Mutability |
 | ---- | ---- | ---- | ---- | ---- |
-| index.[X].elasticsearch.http.auth.type | Authentication type to be used for HTTP(S) access. | String | NONE | LOCAL |
+| index.[X].elasticsearch.http.auth.type | Authentication type to be used for HTTP(S) access. Default value is `NONE`. Other options are `BASIC` and `CUSTOM`. | String | NONE | LOCAL |
```

Comment:
Do we really have to say the default value?

Reply:
The wording here can be improved. Do you think it's better if we say "Available options are `NONE`, `BASIC` and `CUSTOM`"?

Reply:
Sounds better